### PR TITLE
Adding check for included .zip files

### PR DESCRIPTION
**Pull request by kkoppenhaver**
Addresses #134 

Checks for and .zip files present in the repository and throws an error on finding one.

### CODE REVIEW

File: checks/included-plugins.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		$blacklist = array(
+			'\.zip'	=> __( 'Zipped Plugin', 'theme-check' ),
+		);
+
+		checkcount();
+
+		foreach( $blacklist as $file => $reason ) {
+			if ( $filename = preg_grep( '/' . $file . '/', $filenames ) ) {
+				$error = implode( array_unique( $filename ), ' ' );
+				$this->error[] = sprintf( '<span class="tc-lead tc-required">'.__( 'REQUIRED','theme-check').'</span>: '.__( '<strong>Zip file found.</strong> Plugins are not allowed in themes. The zip file found was ', 'theme-check') . '<em>' . __( $error ) . '</em>.' ) ;
```
Comment (grappler) [inline checks/included-plugins.php]: We can't translate variables `__( $error )`. As you are already using `sprintf` you can do `__( 'The zip file found was %s', 'theme-check' )` and then define `$error` as the counterpart to `%s`

**Comment by kkoppenhaver** [comment]
Good call, updated.

**Comment by Otto42** [comment]
You don't need to check the $php_files and $css_files variables. Those are literally just *.php and *.css. They can't be zip files.

**Comment by kkoppenhaver** [comment]
Thanks, updated.

**Comment by Otto42** [comment]
Also, you have this as a REQUIRED check. Has this been discussed amongst the theme-review team? Is this requirement documented in the theme review documentation somewhere?

If not, then it needs to be a RECOMMENDED check instead. Theme Check only implements the existing requirements as created by the review team, it doesn't create new ones. That said, it would be fine as recommended, because including ZIPs in themes makes little sense.

**Comment by grappler** [comment]
Yes, we discussed this before. The requirement this is based on is https://make.wordpress.org/themes/handbook/review/required/#plugins

It does not make sense to have additional zip files in the theme.

**Comment by kkoppenhaver** [comment]
Thanks everyone for the tips and pointers. This was my first PR!

**Comment by grappler** [comment]
@kkoppenhaver I just wanted to let you know this PR has been a success. I has caught a number of themes who included plugins in the theme.

**Comment by kkoppenhaver** [comment]
@grappler That's awesome to hear!

**Pull request merged**